daml-lf/validation/src/main/scala/com/digitalasset/daml/lf/validation/Serializability.scala

```diff
@@ -32,9 +32,26 @@ private[validation] object Serializability {
 
     def checkType(): Unit = checkType(typeToSerialize)
 
+    def isInterface(typ: Type): Boolean = {
+      typ match {
+        case TTyCon(tycon) =>
+          interface.lookupDataType(tycon) match {
+            case Right(DDataType(_, _, cons)) =>
+              cons match {
+                case DataInterface => true
+                case _ => false
+              }
+            case Left(_) => false
```

Comment:
the haskell `isInterface` also looks at the `currentModule`, but iiuc that only happens for serializability _inference_, whereas in scala (iiuc) we only do serializability _validation_